Sprint-2/2-mandatory-debug/1.js

```diff
@@ -9,5 +13,17 @@ function sum(a, b) {
 console.log(`The sum of 10 and 32 is ${sum(10, 32)}`);
 
 // =============> write your explanation here
+//The return statement on its own line causes the function to return immediately, meaning a + b; is never executed.
+//In JavaScript, a return statement without a value implicitly returns undefined.
```

Comment:
Moreover, whatever code you write become invalid  after return keyword. So you should be careful while returning values inside of sum function